subsys/net/Kconfig

```diff
@@ -33,9 +33,10 @@ config NET_BUF_LOG
 
 module = NET_BUF
 module-dep = LOG
+module-def = LOG_LEVEL_WRN
```

Comment:
This is not strictly related to this patch, but overall. This "module-def" bothers me. Nobody knows that it's "default", everyone thinks it's "define".

Reply:
Well, someone may look once, then forget and get confused again. I suggest that we change it and make self-descriptive (in a separate patch of course).

Reply:
I suggest this is done like this for now. If needed there could be a separate PR that changes all the invocations of `module-def` later.